.github/scripts/action.yaml

```diff
@@ -37,11 +37,34 @@ inputs:
 runs:
   using: "composite"
   steps:
+  - uses: actions/setup-python@v5
+    with:
+      python-version: 3.12
+  - name: Copy python script
+    shell: bash
+    run: |
+      set -x
+      # repository root
+      cd ${{ github.action_path }}/../../
+      rsync -rv \
+        --exclude=tests \
+        hermetic_build "${GITHUB_WORKSPACE}"
   - name: Copy shell script
     shell: bash
     run: |
       cd ${{ github.action_path }}
-      cp hermetic_library_generation.sh $GITHUB_WORKSPACE
+      cp hermetic_library_generation.sh "${GITHUB_WORKSPACE}"
+  - name: Install python packages
+    shell: bash
+    run: |
+      cd "${GITHUB_WORKSPACE}"
+      pip install --upgrade pip
+      pip install --require-hashes -r hermetic_build/common/requirements.txt
+      pip install hermetic_build/common
+      pip install --require-hashes -r hermetic_build/library_generation/requirements.txt
+      pip install hermetic_build/library_generation
```

Comment:
I don't think we need to install `library_generation` anymore once we move `ConfigChange` out of this module?

Reply:
Yes. We need to install `library_generation` because config change is still part of it. Once it moves to `common`, we don't need to install it.